pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: shelf
-version: 0.7.0
+version: 0.7.1-dev
 author: Dart Team <misc@dartlang.org>
 description: Web Server Middleware for Dart
 homepage: https://github.com/dart-lang/shelf
 environment:
-  sdk: '>=1.22.0 <2.0.0-dev.infinity'
+  sdk: '>=1.23.0 <2.0.0-dev.infinity'
```

Comment:
too bad we don't allow a different min SDK for `test/` from `lib/`

Reply:
Most of our ecosystem keeps current – not a huge deal